### ISSUE TITLE
Various digFlood suggestions

### ISSUE THREAD

**Issue by ElsaTheHobo** [DFHack/dfhack#1236]
I have 3 suggestions for digFlood that I think would be nice. The first is adding support for inorganics with spaces (aka a lot of the gems). This could be resolved by delimiting inorganics with commas instead of spaces in the command. The second suggestion is being able to set a priority for digFlood's designations. Currently, digFlood sets no designation at all, and my units seem to always take priority over digFlood than other digging designations, even over priority 1. You could possibly add a -priority tag. My third suggestion is a more targeted version of digAll that you can set for all ores or all gems, such as digAllGems[0/1] and digAllOres[0/1]. I am loving digFlood otherwise!

**Comment by lethosor** [comment]
Does using quotes around arguments with spaces in them not work? That should be pretty typical for most commands, and there isn't really a good way to change the delimiter for just digFlood.

For priorities, I just fixed digv (#506), so that should be easy enough to implement.

**Comment by ElsaTheHobo** [comment]
I didn't realize I could use quotes :o I can't test that right now because I went through all the raws like 2 days ago and fixed all ids with spaces lol.

**Comment by lethosor** [comment]
Yeah, there are a couple other command-related features too - see https://dfhack.readthedocs.io/en/latest/docs/Core.html#using-dfhack-commands

**Comment by ElsaTheHobo** [comment]
Fixed up my raws because the fix I used for ID spaces messed up some creature descriptions and stuff- quotes do work! Silly me. That just leaves my two other digFlood suggestions. :D
